src/pages/zh/dev-tools/cli/bubblegum/list-trees.md

```
@@ -0,0 +1,80 @@
+description: 查看所有已保存的 Bubblegum 默克尔树
+---
+
+`mplx bg tree list` 命令显示您创建并本地保存的所有默克尔树。
+
+## 基本用法
+
+```bash
+mplx bg tree list
+```
+
+### 按网络筛选
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)